widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/Chat.kt

```diff
@@ -124,7 +127,8 @@ private fun ChatInputTheme(pallet: ColorPallet): InputTheme? {
                 text = BaseDarkColorTextTheme(this),
                 placeholder = BaseNormalColorTextTheme(pallet),
                 divider = baseShadeColorTheme,
-                sendButton = ButtonTheme(iconColor = primaryColorTheme)
+                sendButton = ButtonTheme(iconColor = primaryColorTheme),
+                mediaButton = ButtonTheme(iconColor = baseNormalColorTheme)
```

Comment:
To double-check, is it correct that you set 'normal' color here? I'm asking because the main goal of this PR is to set  neutral color to the missing places, as far as I understand.

Reply:
This is minor fix.